mopidy_local/commands.py

```diff
@@ -143,19 +147,58 @@ def _check_tracks_in_library(
         return files_to_update, files_in_library
 
     def _find_files_to_scan(
-        self, *, media_dir, file_mtimes, files_in_library, excluded_file_exts
+        self,
+        *,
+        media_dir,
+        file_mtimes,
+        files_in_library,
+        included_file_exts,
+        excluded_file_exts,
     ):
         files_to_update = set()
 
+        def _is_hidden_file(relative_path, file_uri):
+            """Returns True for hidden directories/files: these are ignored"""
+            if any(p.startswith(".") for p in relative_path.parts):
+                logger.debug(f"Skipped {file_uri}: Hidden directory/file")
+                return True
+            else:
+                return False
+
+        def _extension_filters(
+            relative_path, file_uri, included_file_exts, excluded_file_exts
+        ):
+            """Returns True if a file extension appears in the included_file_extensions
+            configuration, or, if this does not exist, if a file extension does not appear
+            in the excluded_file_extensions configuration. Otherwise returns False."""
```

Comment:
I don't think the docstrings add anything in the two inner functions. They read basically as well as the code, and risk becoming outdated with time.